src/environmentd/src/http.rs

```diff
@@ -533,7 +537,8 @@ fn base_router(BaseRouterConfig { profiling }: BaseRouterConfig) -> Router {
             "/hierarchical-memory",
             routing::get(memory::handle_hierarchical_memory),
         )
-        .route("/static/*path", routing::get(root::handle_static));
+        .route("/static/*path", routing::get(root::handle_static))
+        .layer(DefaultBodyLimit::max(MAX_REQUEST_SIZE));
```

Comment:
I looked it up and if you don't specify this, it actually already defaults to 2mb. However, I don't think it hurts to make this explicit.